pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -132,6 +132,11 @@ type PrometheusSpec struct {
 	// +optional
 	ReserveDays int `json:"reserveDays,omitempty"`
 
+	// Configuration for `--storage.tsdb.retention.time`, Units Supported: y, w, d, h, m, s, ms.
```

Comment:
Is this to be true? I found `Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".` in `time.ParseDuration`'s comment

Reply:
OK, these are the units supported in Prom duration, @better0332 we may not validate this with the `time.ParseDuration` function.

Reply:
shoud we use model.ParseDuration() in prom?